### ISSUE TITLE
Why use Rive over Lottie on Android ?

### ISSUE THREAD

**Issue by kaushalyap** [rive-app/rive-android#14]
Seems Rive Android is similar to [Lottie Android](https://github.com/airbnb/lottie-android). So why / when to use Rive over lottie? Specially interested in impact on APK / App Bundle size due to the output file.

**Comment by umberto-sonnino** [comment]
Lottie is a wonderful product, and the possibility to play-back After Effects animations through their library is definitely exciting. But we believe that our runtimes can take things a step further by allowing the designer/developer combo to take full control of the experience. With a runtime built purposefully for Rive, developers can fully control the animation and its hierarchy, optimizing for real-time modifications. 

Rive offers full integration between the [Editor](http://preview-web.rive.app/) and its runtimes, while Lottie/Bodymovin/AE is shared by multiple parties. This is to make sure our features are consistent throughout - in fact, this Android runtime is sharing a core C++ layer with its siblings (those are [Web](https://github.com/rive-app/rive-wasm) and [iOS](https://github.com/rive-app/rive-ios)).
Moreover, the runtimes and the exported Rive files are tiny, often smaller than Lottie.

To sum it up: Rive is more of a runtime engine, solving for dynamic properties that can be driven by designed animations or procedural code. it's closer to what you'd find in a game engine and supports richer interactions.

There's an example of integrating a pull-to-refresh animation within your Android app [on our blog](https://blog.rive.app/pull-to-refresh-android/) demonstrates how everything is updated and controlled in real-time in code. We're hoping to ship more examples like this in the future..!

**Comment by kaushalyap** [comment]
@umberto-sonnino Thanks for the detailed explanation.

**Comment by MuhHarisPK** [comment]
can you give me a solution for reducing the APK/Bundle size?
I am experiencing an issue with my Android project where the project size has increased significantly after integrating the Rive SDK (formerly known as Flare). Prior to integrating the Rive SDK, my project size was within an acceptable range, but after adding the SDK, the size has increased beyond expectations.
